### PR TITLE
Only build with `--direct_html`

### PR DESCRIPTION
**Pull request by nik9000**
This removes support for building docs without direct_html. You no
longer need to specify `--direct_html` on the command line or
`direct_html` in conf.yaml for it ot build with direct_html.

This doesn't drop the configuration from conf.yaml or the change
tracker or all of the files that we need to build with docbook. We'll do
that in a followup.

### CODE REVIEW

**Comment by nik9000** [comment]
This needs to wait on #1559.

File: build_docs.pl [hunk 1]
```diff
@@ -69,6 +69,12 @@ BEGIN
 forever in honor of our success but otherwise doesn't do anything.
 MSG
 }
+if ( $Opts->{direct_html} ) {
+   say <<MSG
+The direct_html migration is complete! --direct_html will emit this message
+forever in honor of our success but otherwise doesn't do anything.
+MSG
+}
```
Comment (JoshMock) [inline build_docs.pl]: 😍

Comment (nik9000) [reply build_docs.pl]: I don't want to drop the parameter or else it'll break everyone that added it1

**Comment by nik9000** [comment]
@elasticmachine, run elasticsearch-ci/docs rebuild

**Pull request merged**